src/Swashbuckle.AspNetCore.SwaggerGen/XmlComments/XmlCommentsExampleHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Text.Json;
+using Microsoft.OpenApi.Any;
+using Microsoft.OpenApi.Models;
+
+namespace Swashbuckle.AspNetCore.SwaggerGen
+{
+    public static class XmlCommentsExampleHelper
+    {
+        public static IOpenApiAny Create(
+            SchemaRepository schemaRepository,
+            OpenApiSchema schema,
+            string exampleString)
+        {
+            var isStringType =
+                (schema?.ResolveType(schemaRepository) == "string") &&
+                !exampleString.Equals("null");
```

Comment:
The static `string.Equals()` method is robust to nulls that the instance method.

Reply:
done